### PR TITLE
simplify daily / monthly build

### PR DESCRIPTION
**Pull request by rkpagadala**
Currently we builds 2 sets of images one of which points to gcr.io/istio-testing and another to docker.io/istio This has been a great source of confusion and complexity.

With this change
1) Build parameters are passed from Airflow to GCB via gcb_env.sh which contains exported parameters
2) Makes testing release changes from personal repo possible
3)  Only one set of artifacts are created
Daily Builds
Docker images are pushed to gcr.io/istio-release, istioctl and the yaml/helm files also point to gcr.io/istio-release. This set of artifacts are then tested and published to
https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/

Monthly Builds
Docker images are pushed to docker.io/istio, istioctl and the yaml/helm files also point to docker.io/istio. This set of artifacts are then tested and published to github as a release

### CODE REVIEW

File: release/gcb/generate_manifest.sh [hunk 1]
```diff
@@ -124,7 +112,7 @@ function get_later_sha_timestamp() {
   MANIFEST_FILE=$2
 
   local SHA_MFEST
-  SHA_MFEST=$(grep istio/istio "$MANIFEST_FILE" | sed 's/.*istio. revision=.//' | sed 's/".*//')
+  SHA_MFEST=$(grep "istio\"" "$MANIFEST_FILE" | sed 's/.*istio. revision=.//' | sed 's/".*//')
```
Comment (hklai) [inline release/gcb/generate_manifest.sh]: ?

File: release/gcb/generate_manifest.sh [hunk 2]
```diff
@@ -223,75 +208,62 @@ pushd istio
...
-  case "${arg}" in
-    a) CHECK_GREEN_SHA_AGE="${OPTARG}";;
-    b) BRANCH="${OPTARG}";;
-    c) COMMIT="${OPTARG}";;
-    r) GCS_RELEASE_TOOLS_PATH="gs://${OPTARG}";;
-    v) VERIFY_CONSISTENCY="${OPTARG}";;
-    *) usage;;
-  esac
-done
+[[ -z "${CB_BRANCH}"                 ]] && usage
```
Comment (hklai) [inline release/gcb/generate_manifest.sh]: remove?

File: release/gcb/gcb_lib.sh [hunk 3]
```diff
@@ -15,15 +15,22 @@
...
     pushd "${TEST_INFRA_DIR}" || exit 1
-     bazel build //toolbox/githubctl
-     githubctl="${TEST_INFRA_DIR}/bazel-bin/toolbox/githubctl/linux_amd64_stripped/githubctl" 
+     if [[ -f "/workspace/githubctl" ]]; then
+       githubctl="/workspace/githubctl"
+       ls -l "$githubctl"
+       chmod +x "$githubctl"
+       ls -l "$githubctl"
+     else
+       bazel build //toolbox/githubctl
```
[the remark this answers is not in the document]
Comment (rkpagadala) [reply release/gcb/gcb_lib.sh]: only remaining issue

**Comment by rkpagadala** [comment]
@hklai PTAL code is tested

**Pull request merged**